File: POO/Python/uberVan.py
#/usr/bin/env python3.8
from carrito import Car
class UberVan(Car):
    typeCarAccepted = str
    seatsMaterial = str
    def __init__(self, license, driver, typeCarAccepted, seatsMaterial):
        super().__init__(license, driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial
    def printDataCar(self): #Este 'self' es muy importante
        print(self.seatsMaterial)

File: POO/PHP/uberVan.php
<?php
use function PHPSTORM_META\map;
require_once("car.php");
class UberVan extends Car{
        public $seatsMaterial;
        public $typeCarAccepted;
        public function __construct($license, $driver, $seatsMaterial, $typeCarAccepted){
            parent::__construct($license, $driver);
            $this->seatsMaterial = $seatsMaterial;
            $this->typeCarAccepted = $typeCarAccepted;
        }
    }
?>